core-api/src/worker.rs

```diff
@@ -90,6 +92,11 @@ impl WorkerConfigBuilder {
         if self.max_concurrent_wft_polls == Some(0) {
             return Err("`max_concurrent_wft_polls` must be at least 1".to_owned());
         }
+        if self.max_outstanding_workflow_tasks > self.max_cached_workflows {
+            return Err(
+                "`max_outstanding_workflow_tasks` cannot exceed `max_cached_workflows`".to_owned(),
+            );
+        }
```

Comment:
The only issue with this error string is that it uses the rust config property names which are different than the lang ones so it might confuse some users.

Reply:
I don't think there's any good resolution to that though, besides lang intercepting the error.

Reply:
Lang would need to intercept this specific error string.
We could just rename this from "outstanding" to "concurrent" in Core and it would be good enough.

Reply:
FWIW, in Python we also changed `outstanding` to `concurrent`. As for exact var naming, using English words instead of var names can help alleviate those concerns (e.g. just remove the backticks and underscores).

Reply:
Just used plain English.